drupal/recipes/wunder_next_setup/config/next.next_entity_type_config.node.article.yml

```diff
@@ -7,7 +7,7 @@ configuration:
   sites:
     frontend: frontend
   field_name: {  }
-revalidator: ''
+revalidator: path
 revalidator_configuration:
-  revalidate_page: null
-  additional_paths: null
+  revalidate_page: true
+  additional_paths: "/\r\n/fi\r\n/en\r\n/sv"
```

Comment:
this tells drupal to revalidate the frontpage path for all languages when an article is created (this because in our site at the moment the frontpage displays a list of articles)

This is a poor's man cache-tags kind of idea...